watchtower-plugin/src/main.rs

```diff
@@ -483,10 +510,7 @@ async fn on_commitment_revocation(
             state.add_pending_appointment(tower_id, &appointment);
 
             if status.is_temporary_unreachable() {
-                state
-                    .unreachable_towers
-                    .send((tower_id, appointment.locator))
-                    .unwrap();
+                send_to_retrier(&state, tower_id, appointment.locator);
             }
```

Comment:
@sr-gi I think it's gonna be clearer when the whole context is annotated.
We will only send to retrier this fresh revocation data if it's temporary unreachable,
some retriers are in sub error state while running, these ones will miss this fresh data.

Reply:
Ohh gotcha, I was looking at another line. Yeah, this should be `if !status.is_unreachable` to account for both.